server/aspects/rules/python/python_info.bzl.template

```diff
@@ -1,20 +1,23 @@
-#if( $pythonEnabled == "true" && $bazel8OrAbove == "true" )
+#if( $pythonEnabled == "true" )
+#if( $bazel8OrAbove == "true")
 load("@rules_python//python:defs.bzl", "PyInfo", "PyRuntimeInfo")
 #end
+load("@rules_python//python:defs.bzl", PyInfoRulesPython = "PyInfo", PyRuntimeInfoRulesPython = "PyRuntimeInfo")
```

Comment:
Are you sure this is correct? You load the same PyInfo twice if `bazel8OrAbove == "true"`

Reply:
Loading again in this case would be ok since it's under a different alias (originally I was thinking of trying to avoid adding to much additional logic to the conditionals in the file).

However I found that this approach does not properly accommodate Bazel versions <7, which could fail if the workspace did not explicitly include rules_python.  I made some updates to handle this case and make the rest of the logic clearer.